docs/source/checks/tabular/model_evaluation/plot_unused_features.py

```diff
@@ -62,5 +62,5 @@
 # We can define a condition that enforces that number of unused features with high variance is not greater than a given
 # amount, the default is 5.
 check = UnusedFeatures().add_condition_number_of_high_variance_unused_features_less_or_equal(5)
-result = check.run(test_ds, model)
+result = check.run(train_ds, test_ds, model)
```

Comment:
Following @nirhutnik's previous comment, unused features is a SingleDatasetCheck, therefore it should accept only one dataset.

In the context of the check, a more beneficial scenario would be to run it on the "test" dataset rather than on the train, therefore I suggest leaving the file as was originally (in lines 53 and 56 as well)